test/webui/components/PakageSidebar/maintainers.spec.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Maintainers component
+ */
+
+import React from 'react';
+import { mount } from 'enzyme';
+import Maintainers from '../../../../src/webui/src/components/PackageSidebar/modules/Maintainers';
+import { packageMeta } from '../store/packageMeta';
+
+console.error = jest.fn();
+
+describe('<PackageSidebar /> : <Maintainers />', () => {
+  let wrapper;
+  let instance;
+
+  beforeEach(() => {
+    wrapper = mount(<Maintainers packageMeta={packageMeta} />);
+    instance = wrapper.instance();
+  });
+
+  it('should match with the props', () => {
+    expect(wrapper.props().packageMeta).toEqual(packageMeta);
+    expect(wrapper.html()).toMatchSnapshot();
+  });
+
+  it('author shoule be equal to Alex Kocharin', () => {
+    expect(instance.author).toEqual({
+      avatar:
+        'https://www.gravatar.com/avatar/a5a236ba477ee98908600c40cda74f4a',
+      email: 'alex@kocharin.ru',
+      name: 'Alex Kocharin'
+    });
+  });
+
+  it('should get all the contributors with false for showAllContributors', () => {
+    expect(instance.showAllContributors).toBeFalsy();
+  });
+
+  it('should get unique contributors', () => {
+    const result = [
+      {
+        avatar:
+          'https://www.gravatar.com/avatar/4ef03c2bf8d8689527903212d96fb45b',
+        email: 'chocolatey030@gmail.com',
```

Comment:
please, let use fake emails ... we do not want piss off someone with smap :)

Reply:
I'll change emails.